metadata-ingestion/src/datahub/ingestion/source/usage/bigquery_usage.py

```diff
@@ -370,7 +370,9 @@ def from_entry(cls, entry: AuditLogEntry) -> "QueryEvent":
     def get_missing_key_exported_bigquery_audit_metadata(
         row: BigQueryAuditMetadata,
     ) -> Optional[str]:
-        return get_first_missing_key_any(row, ["timestamp", "protoPayload", "metadata"])
+        return get_first_missing_key_any(
+            row._xxx_field_to_index, ["timestamp", "protoPayload", "metadata"]
```

Comment:
Is this a typo? Don't see any member called `_xxx_field_to_index`.

Reply:
You don't see any member because this is a `Row` returned by bigquery. See https://github.com/datahub-project/datahub/issues/4529. `BigQueryAuditMetadata`  is `Any` in this source not a specific class.

Reply:
See https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/usage/bigquery_usage.py#L48

I understand this should probably be fixed. But currently fixing lineage related bugs so not touching that right now.

Reply:
Please add comments + a link to the documentation.